app/client/templates/views/send.js

```diff
@@ -101,38 +101,49 @@ Template['views_send'].onCreated(function(){
     TemplateVar.set('amount', '0');
     TemplateVar.set('estimatedGas', 300000);
     TemplateVar.set('sendAll', false);
+    
+    // Deploy contract
+    if(FlowRouter.getRouteName() === 'deployContract') {
+        TemplateVar.set('selectedAction', 'deploy-contract');
```

Comment:
This is the PR link: https://github.com/ethereum/meteor-dapp-wallet/pull/361
@luclu would you elaborate how that works?